x/meicdp/handler.go

```diff
@@ -19,23 +23,122 @@ func NewHandler(keeper Keeper) sdk.Handler {
 		case channeltypes.MsgPacket:
 			var responseData oracle.OracleResponsePacketData
 			if err := types.ModuleCdc.UnmarshalJSON(msg.GetData(), &responseData); err == nil {
-				fmt.Println("I GOT DATA", responseData.Result, responseData.ResolveTime)
-				// handleOraclePacket(ctx, keeper, responseData)
+				err := handleOracleRespondPacketData(ctx, keeper, responseData)
```

Comment:
Will this fail the whole transaction if an error occurs?

Reply:
yes, this err will fail the whole transaction and didn't block other action